src/stactools/naip/stac.py

```diff
@@ -108,7 +110,7 @@ def create_item(
     """
 
     with rio.open(cog_href) as ds:
-        gsd = ds.res[0]
+        gsd = round(ds.res[0], 1)
```

Comment:
rounds resolution to 0.6 or 1.0 instead of slightly different decimals reported in geotiff metadata